webapp/src/styles/theme.tsx

```diff
@@ -23,7 +23,7 @@ const customTheme: Theme = createTheme({
       default: "#fafafa", // TODO awaiting design input.
     },
     primary: {
-      light: "#3C3A57",
+      light: "#d5d1e3",
```

Comment:
That's a tiny detail, but would you mind using uppercase like the other colors defined here?
```suggestion
      light: "#D5D1E3",
```

Reply:
okay sure going forward